draft-ietf-capport-api.md

```diff
@@ -127,7 +127,42 @@ Information passed in this protocol may include a user's personal information, s
 
 # IANA Considerations
 
-TBD: None
+This document registers the media type for Captive Portal API JSON text, "application/captive+json".
+
+Type name: application
```

Comment:
You might have more luck with formatting like:

```
label:

: value

label:

: value
```

I find that to be the best way to do these lists.

Reply:
Ah, thanks! Updated.